include/IGUIComboBox.h

```diff
@@ -52,6 +52,10 @@ namespace gui
 		//! Sets the selected item. Set this to -1 if no item should be selected
 		virtual void setSelected(s32 idx) = 0;
 
+		//! Sets the selected item and emits a change event.
+		/** Set this to -1 if no item should be selected */
+		virtual void setAndSendSelected(s32 idx) = 0;

```

Comment:
why doesn't `setSelected` just do this?
I'm not saying you have to change the PR in that fashion, but it would be more logical.

Reply:
I agree that it would be better to have `setSelected` do this, but `setSelected` is used a lot in `CGUIComboBox.cpp` and I'd prefer not to rewrite that file.